classes/Instructors_List.php

```diff
@@ -207,6 +207,14 @@ public static function update_instructors( $status, $user_ids ): bool {
 				self::remove_instructor_role( $instructor_id, $status );
 			}
 		}
+		if ( 'reject' === $status ) {
+			$arr = explode( ',', $user_ids );
+			foreach ( $arr as $instructor_id ) {
+				$instructor_id = (int) sanitize_text_field( $instructor_id );
+				self::remove_instructor_role( $instructor_id, $status );
```

Comment:
Remove the hooks from this method and make action specific method by wrapping this method.